docs/submitting-applications.md

```diff
@@ -143,6 +143,9 @@ The master URL passed to Spark can be in one of the following formats:
 <tr><td> <code>spark://HOST:PORT</code> </td><td> Connect to the given <a href="spark-standalone.html">Spark standalone
         cluster</a> master. The port must be whichever one your master is configured to use, which is 7077 by default.
 </td></tr>
+<tr><td> <code>spark://HOST1:PORT1,HOST2:PORT2</code> </td><td> Connect to the given <a href="spark-standalone.html#standby-masters-with-zookeeper">Spark standalone
```

Comment:
I came across a situation where the developers using the spark driver believed that it was not possible to accept hosts in the mentioned format because it wasn't mentioned in the docs, so I found it useful to add the detail and submit a PR.
Cheers!